opencti-platform/opencti-front/src/private/components/data/ingestionTaxii/IngestionTaxiiEdition.jsx

```diff
@@ -39,7 +39,16 @@ const ingestionTaxiiValidation = (t) => Yup.object().shape({
   cert: Yup.string().nullable(),
   key: Yup.string().nullable(),
   ca: Yup.string().nullable(),
-  user_id: Yup.object().nullable(),
+  user_id: Yup.lazy((value) => {
+    switch (typeof value) {
+      case 'object':
+        return Yup.object().nullable(); // schema for object
+      case 'string':
+        return Yup.string().nullable();
+      default:
+        return Yup.mixed();
+    }
+  }),
```

Comment:
I'm wondering if we could use Yup.mixed().nullable() :

`user_id: Yup.mixed().nullable(),`

I tested it and it works, I can see it used in other components, but I'm not sure of the best practice in this case. What do you think @Kedae ?

Also, we have the same issue for RSS and sync (SyncEdition, IngestionRSSEdition)

Reply:
I think the interest of lazy with a switch is to have a list of constrained possibilities.

In that case, since we have a default as mixed, a global mixed should be enough

Reply:
Done :)